.gitignore

```diff
@@ -1,3 +1,5 @@
 .env
 
-*.csv
\ No newline at end of file
+*.csv
+
```

Comment:
Do we need all these newlines?

Reply:
I think it is good to keep these files separated (keeps different file types organised), I would keep it.